go.sum

```diff
@@ -251,12 +236,13 @@ k8s.io/klog v0.4.0 h1:lCJCxf/LIowc2IGS9TPjWDyXY4nOmdGdfcwwDQCOURQ=
 k8s.io/klog v0.4.0/go.mod h1:4Bi6QPql/J/LkTDqv7R/cd3hPo4k2DG6Ptcz060Ez5I=
 k8s.io/kube-openapi v0.0.0-20180731170545-e3762e86a74c h1:3KSCztE7gPitlZmWbNwue/2U0YruD65DqX3INopDAQM=
 k8s.io/kube-openapi v0.0.0-20180731170545-e3762e86a74c/go.mod h1:BXM9ceUBTj2QnfH2MK1odQs778ajze1RxcmP6S8RVVc=
-k8s.io/utils v0.0.0-20190506122338-8fab8cb257d5 h1:VBM/0P5TWxwk+Nw6Z+lAw3DKgO76g90ETOiA6rfLV1Y=
 k8s.io/utils v0.0.0-20190506122338-8fab8cb257d5/go.mod h1:sZAwmy6armz5eXlNoLmJcl4F1QuKu7sr+mFQ0byX7Ew=
 k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a h1:uy5HAgt4Ha5rEMbhZA+aM1j2cq5LmR6LQ71EYC2sVH4=
 k8s.io/utils v0.0.0-20190809000727-6c36bc71fc4a/go.mod h1:sZAwmy6armz5eXlNoLmJcl4F1QuKu7sr+mFQ0byX7Ew=
-sigs.k8s.io/cluster-api v0.0.0-20190829154355-58b8460b80c5 h1:iF/85F2lFHJLDxJk2KQC1Yps51bAHCpehNryTvSr1vM=
-sigs.k8s.io/cluster-api v0.0.0-20190829154355-58b8460b80c5/go.mod h1:5ODrAYsZYC0gHyqu20VcqnD3uLXaqzFN9HzBwYAAKNs=
+sigs.k8s.io/cluster-api v0.0.0-20190829222858-82b282b06aaf h1:YkMawPF8j53aoMqpDFdaCXd0jYX4nOm/33677WLUfWQ=
+sigs.k8s.io/cluster-api v0.0.0-20190829222858-82b282b06aaf/go.mod h1:5ODrAYsZYC0gHyqu20VcqnD3uLXaqzFN9HzBwYAAKNs=
+sigs.k8s.io/cluster-api v0.0.0-20190830155554-d495192f76e0 h1:qV0Rvi06IYwPwlmvCt8F2TTlp62Cn7hZCmI1CTtymFM=
+sigs.k8s.io/cluster-api v0.0.0-20190830155554-d495192f76e0/go.mod h1:5ODrAYsZYC0gHyqu20VcqnD3uLXaqzFN9HzBwYAAKNs=
```

Comment:
Did `go mod tidy` leave 2 versions of CAPI here?

Reply:
Fixed in latest push